packages/codemirror/src/editor.ts

```diff
@@ -166,7 +166,7 @@ export class CodeMirrorEditor implements CodeEditor.IEditor {
     // Turn off paste handling in codemirror since sometimes we want to
     // replace it with our own.
     editor.on('paste', (instance: CodeMirror.Editor, event: any) => {
-      if (!this._config['handlePaste']) {
+      if (this._config['handlePaste'] !== undefined && !this._config['handlePaste']) {
```

Comment:
If `this._config['handlePaste'] === undefined`, should we also ignore?

In that case, keeping the condition as `!this._config['handlePaste']` may cover both if it's undefined and if it's false, since they are both falsey.

Reply:
I think if its undefined then we want codemirror to handle pasting. Because then if someone doesn't define it they will be expected to implement pasting themselves.